content/microcopy/box_ui_elements.yml

```diff
@@ -0,0 +1,19 @@
+---
+demoTiles:
+  contentExplorer:
+    title: Content Explorer
+    description: Embed a folder view of content stored in Box in your
```

Comment:
```suggestion
    description: Embed a folder view of content you store in Box in your
```

Reply:
It's not necessary folder that this person stores, eg this can be enterprise folder, so I'd stick with the current form.